content/atoms.html

```diff
@@ -1,133 +1,222 @@
+  </head>
+
+  <body>
+    <header class=main-header>
+      <h1>
+        <a href=index.html>Neomad - Styleguide</a>
+      </h1>
+      <nav>
+        <a href=atoms.html>Atoms</a>
+        <a href=molecules.html>Molecules</a>
+        <a href=header-footer.html>Header-Footer</a>
+      </nav>
+    </header>
+
+    <main class=st-main>
+      <h1 class=st-title>Atoms</h1>
+      <section class=st-section>
+
+        <article class=st-article>
+          <h2 class=st-title>Colors</h2>
+          
+          <div class=st-container>
+            <ul class=st-colors>
+              <li>
+                <span style=background-color:#3dd07d></span>
+                <span>--primary-color</span>
+                <span>#3dd07d;</span>
+              </li>
+              <li>
+                <span style=background-color:#4e8></span>
+                <span>--secondary-color</span>
+                <span>#4e8;</span>
+              </li>
+              <li>
+                <span style=background-color:#000></span>
+                <span>--black</span>
+                <span>#000;</span>
+              </li>
+              <li>
+                <span style=background-color:#fff></span>
+                <span>--white</span>
+                <span>#fff;</span>
+              </li>
+              <li>
+                <span style=background-color:#fefefe></span>
+                <span>--primary-bg-color</span>
+                <span>#fefefe;</span>
+              </li>
+              <li>
+                <span style=background-color:#fafafa></span>
+                <span>--secondary-bg-color</span>
+                <span>#fafafa;</span>
+              </li>
+            </ul>
+          </div>
+        </article>
+
+        <article class=st-article>
+          <h2 class=st-title>Fonts</h2>
+
+          <div class=st-container>
+            <ul>
+              <li class=st-font-title>--head-font: Poppins</li>
+              <li class=st-font-content>--content-font: Maven Pro</li>
+              <li class=st-font-blog>--blog-font: Merryweather</li>
+            </ul>
+          </div>
+        </article>
+
+        <article class=st-article>
+          <h2 class=st-title>Headings</h2>
+
+          <div class=st-container>
+            <h1 class=st-font-title>Default Title level 1</h1>
+            <h2 class=st-font-title>Default Title level 2</h2>
+            <h3 class=st-font-title>Default Title level 3</h3>
+          </div>
+        </article>
+
+        <article class=st-article>
+          <h2 class=st-title>Text content</h2>
+
+          <div class=st-container>
+            <p>The <a href=#>a element</a> and <a href=# target=_blank>external a element</a> examples</p>
+            <p>The <abbr>abbr element</abbr> and an <abbr title=Abbreviation>abbr</abbr> element with title examples</p>
+            <p>The <acronym title=A Cowboy Ran One New York Marathon>ACRONYM</acronym> element example</p>
+            <p>The <b>b element</b> example</p>
```

Comment:
deprecated for strong

Reply:
what? since when? https://developer.mozilla.org/fr/docs/Web/HTML/Element/strong